'jquery.i18n' is not in the npm registry.

When I try to install with NPM I get the following error:
> Not Found
> 
> 'jquery.i18n' is not in the npm registry.
> You should bug the author to publish it

This issue exists to "bug the author to publish it"

I tried publishing. But it has name conflicts
```
npm notice
npm ERR! publish Failed PUT 403
npm ERR! code E403
npm ERR! Package name too similar to existing packages; 
```

--- a/package.json
+++ b/package.json
@@ -1,5 +1,5 @@
 {
-  "name": "jquery.i18n",
+  "name": "@wikimedia/jquery.i18n",
   "version": "1.0.5",
   "description": "jQuery based internationalization library",
   "homepage": "https://github.com/wikimedia/jquery.i18n",

